### ISSUE TITLE
Version 3.6 / 4.0

### ISSUE THREAD

**Issue by josepmy** [leukbaars/DreamUV#39]
Is there any work in progress to update DreamUV to version 3.6 or 4.0?

**Comment by Kozzren** [comment]
I really hope this conitues to work too, I cant live without this addon anymore.

**Comment by unfa** [comment]
I don't think the author is active and have been for the past few years. So I think we need to find a way to maintain this plug-in ourselves...

**Comment by josepmy** [comment]
@unfa @Kozzren by the way the plugin seems is working on 3.6 and 4.0

**Comment by Kozzren** [comment]
> @unfa @Kozzren by the way the plugin seems is working on 3.6 and 4.0

Yeah I know thats why i said continue to work, cause without how fast blender updates yearly its only a matter of time.

**Comment by leukbaars** [comment]
I am still active and use this tool every day in my professional career. It's been a bit exhausting to keep up to date with blender. I only recently finally switched from using 3.5. Tool is updated to 4.1 now and I added a bunch of features for working with trim sheets, box mapping and multiple uv sets. Let me know if you find any problems!

**Comment by unfa** [comment]
Oh my! I'm so glad to see you again, @leukbaars ! I was worried we might never see you again!

On a side note (sorry for going offtopic):

I agree that Blender moves so fast, it's hard to build a solid production environment on it. I managed to botch DreamUV to work with 4.1, but many add-ons that would be so useful in game asset production are just left behind. And it's hard to blame volunteers making open-source tools for not keeping up. I wish Blender had some backwards compatibility layer for add-ons...

Do you happen to have a solution for batch exporting assets to GLTF from Blender? It seems FBX has better support, but add-ons like FBX Bundle or  Bundle Exporter got outdated too.

**Comment by guategeek** [comment]
Haven't tested this with 4.1 yet but this is the batch export plugin I have used, it has a lot of really handy features. And you can create export presets for any format Blender supports: https://github.com/Takanu/Capsule

Sent with Unibox

**Comment by Kozzren** [comment]
> I am still active and use this tool every day in my professional career. It's been a bit exhausting to keep up to date with blender. I only recently finally switched from using 3.5. Tool is updated to 4.1 now and I added a bunch of features for working with trim sheets, box mapping and multiple uv sets. Let me know if you find any problems!

im so glad your updating this still, im a track maker for a couple of games and i cannot live without this addon!
